.github/pull_request_template.md

```diff
@@ -1,14 +1,9 @@
 **This pull request makes the following changes:**
-* Fixes issue code-collabo/node-mongo-cli#issueNumberHere 
+* Fixes issue #issueNumberHere 
```

Comment:
Same thing here too. Change this line to:
* Fixes issue code-collabo/repo-name-here#issueNumberHere

Reply:
I have made the change